hack/e2e.sh

```diff
@@ -19,57 +19,26 @@ source "${SCRIPT_DIR}/libe2e.sh"
 helm delete --purge "${RELEASE_NAME}" || true
 kube_delete_namespace_and_wait "${USER_NAMESPACE}"
 
-echo "Waiting up to 5 minutes for Kubernetes to be ready..."
-retry TIMEOUT=600 kubectl get nodes
-
-echo "Waiting for tiller to be ready..."
-retry TIMEOUT=60 helm version
-
 echo "Installing navigator..."
 helm install --wait --name "${RELEASE_NAME}" contrib/charts/navigator \
         --set apiserver.image.pullPolicy=Never \
         --set controller.image.pullPolicy=Never
 
-# Wait for navigator pods to be running
+# Wait for navigator API to be ready
 function navigator_ready() {
-    local replica_count_controller=$(
-        kubectl get deployment ${RELEASE_NAME}-navigator-controller \
-                --output 'jsonpath={.status.readyReplicas}' || true)
-    if [[ "${replica_count_controller}" -eq 0 ]]; then
-        return 1
-    fi
-    local replica_count_apiserver=$(
-        kubectl get deployment ${RELEASE_NAME}-navigator-apiserver \
-                --output 'jsonpath={.status.readyReplicas}' || true)
-    if [[ "${replica_count_apiserver}" -eq 0 ]]; then
-        return 1
+    if kubectl api-versions | grep 'navigator.jetstack.io'; then
+        return 0
     fi
     return 0
 }
 
-if ! retry navigator_ready; then
-        kubectl get pods --all-namespaces
-        kubectl describe deploy
-        kubectl describe pod
-        exit 1
+echo "Waiting up to 10 minutes for Navigator to be ready..."
+if ! retry TIMEOUT=600 navigator_ready; then
```

Comment:
I'd class the API taking 10 minutes to register as a bug and not something we should accept - can we remove this change from this PR?

Happy to lend a hand debugging what's going on your end to cause those issues though. Let's discuss and work through it on Slack.

Reply:
It seems to be related to RBAC.
If I start minikube without rbac enabled and disable rbac in the navigator chart, the Navigator API becomes ready within 3 minutes.